model/Check/Instance/MathJaxCheck.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoSystemStatus\model\Check\Instance;
+
+use common_report_Report as Report;
+use common_ext_ExtensionException;
+use oat\taoSystemStatus\model\Check\AbstractCheck;
+
+/**
+ * Class MathJaxCheck
+ * @package oat\taoSystemStatus\model\Check\System
+ * @author Aleksej Tikhanovich, <aleksej@taotesting.com>
+ */
+class MathJaxCheck extends AbstractCheck
+{
+    private const MATH_JAZ_FOLDER_PREFIX = 'views/js/mathjax';
```

Comment:
There is a typo : *_JAZ*
Please remove `private` as it is available only from PHP 7.1